index.bs

```diff
@@ -270,9 +270,10 @@ A [=/screen=] has a <dfn>device pixel ratio</dfn>, similar to {{Window}}'s {{Win
 
 A [=/screen=] has an <dfn>orientation</dfn>, which is described in [[screen-orientation]].
 
-A [=/screen=] has a <dfn>label</dfn>, which is a string that meaningfully describes the screen to a user.
+A [=/screen=] has a <dfn>label</dfn>, which is a string that meaningfully describes the screen to a user to help them identify and tell the difference between screens.
```

Comment:
nit: "tell the difference between" -> "differentiate"

Reply:
Done